<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CanMessageDlgClass</class>
 <widget class="QDialog" name="CanMessageDlgClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New transmit message</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout" columnstretch="1,1,0,0">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Id (Hex):</string>
       </property>
       <property name="buddy">
        <cstring>lineEditId</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Length:</string>
       </property>
       <property name="textFormat">
        <enum>Qt::LogText</enum>
       </property>
       <property name="buddy">
        <cstring>spinBoxDlc</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Data (0..7):</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="lineEditId">
       <property name="inputMask">
        <string>HHH; </string>
       </property>
       <property name="text">
        <string>000</string>
       </property>
       <property name="cursorMoveStyle">
        <enum>Qt::VisualMoveStyle</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBoxDlc">
       <property name="maximum">
        <number>8</number>
       </property>
       <property name="value">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEditData0">
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="inputMask">
          <string>HH; </string>
         </property>
         <property name="text">
          <string>00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditData1">
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="inputMask">
          <string>HH; </string>
         </property>
         <property name="text">
          <string>00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditData2">
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="inputMask">
          <string>HH; </string>
         </property>
         <property name="text">
          <string>00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditData3">
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="inputMask">
          <string>HH; </string>
         </property>
         <property name="text">
          <string>00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditData4">
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="inputMask">
          <string>HH; </string>
         </property>
         <property name="text">
          <string>00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditData5">
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="inputMask">
          <string>HH; </string>
         </property>
         <property name="text">
          <string>00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditData6">
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="inputMask">
          <string>HH; </string>
         </property>
         <property name="text">
          <string>00</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditData7">
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="inputMask">
          <string>HH; </string>
         </property>
         <property name="text">
          <string>00</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Period [ms]:</string>
       </property>
       <property name="buddy">
        <cstring>lineEditPeriod</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLineEdit" name="lineEditPeriod">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QCheckBox" name="checkBoxExtended">
       <property name="text">
        <string>Extended Frame</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QCheckBox" name="checkBoxRtr">
       <property name="text">
        <string>Remote Request</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CanMessageDlgClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CanMessageDlgClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
